apps/web/nuxt.config.ts

```diff
@@ -174,7 +175,7 @@ export default defineNuxtConfig({
   },
   runtimeConfig: {
     public: {
-      apiUrl: process.env.API_URL ?? 'http://localhost:8181',
+      apiUrl: removeTrailingSlash(process.env.API_URL),
```

Comment:
would rename that function to something like `validateApiUrl` oder `getApiUrl` and if we make it validateApiUrl I would extract the localhost:8181 address back to were it was before.